lib/validate-selectors.js

```diff
@@ -2,6 +2,40 @@ var listSequences = require('./list-sequences');
 var shouldIgnoreRule = require('./should-ignore-rule');
 var shouldIgnoreSelector = require('./should-ignore-selector');
 var toInterpoloatedRegexp = require('./to-interpolated-regexp');
+var resolveNestedSelector = require('postcss-resolve-nested-selector');
+
+function isTopLevel(node) {
+  return node.type === 'atrule' || node.type === 'root';
```

Comment:
Are at-rules always "top level"? What if you're nesting an at-rule inside a rule, e.g.

```scss
.foo {
  &:hover { color: pink; }
  @media (min-width: 400px) {
    &:hover { color: orange; }
  }
}
```